org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -42,30 +40,45 @@
   // This is an error code we made up to signal that available disk space is less than 50%.
   // It may be used in tracking on Cloud Watch.
   private static final int LOW_DISK_SPACE_ERROR_CODE = 50;
+  private static final String CONTENT_BUCKET_NAME = System.getenv("CONTENT_BUCKET_NAME");
+  private static final String CONTENT_BUCKET_URL = System.getenv("CONTENT_BUCKET_URL");
+  private static final String API_ENDPOINT = System.getenv("API_ENDPOINT");
+  private final AmazonApiGatewayManagementApi API_CLIENT;
+  private final AmazonSQS SQS_CLIENT;
+  private final AmazonS3 S3_CLIENT;
```

Comment:
nit: lowercase names? (`apiClient`,  etc)

Reply:
Oh good call. Actually, I'm realizing these should probably be static since they are configured based on environment variables and won't change unless they're on a different environment.